### PR TITLE
 feat(*): added semantic-release and packaged to PyPI

### PR DESCRIPTION
**Pull request by mostaphaRoudsari**
I followed @AntoineDao's lead and made similar changes that he made for uwg here! Hopefully it will pass the test and builds with no issues. Let's see how it goes. 💭

PS: This build is now on PyPI!
https://pypi.org/project/lbt-ladybug/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,3 +13,7 @@ script:
 after_success:
   - coverage report
   - coveralls
+  - git config --global user.name "semantic-release (via TravisCI)"
```
Comment (AntoineDao) [inline .travis.yml]:
```suggestion
  - git config --global user.name "ladybugbot"
  - git config --global user.email "ladybugbot@whateveremailprovider.com"
```

Comment (AntoineDao) [reply .travis.yml]: I'd recommend using the ladybugbot user here as that is what will set the user releasing code. The GH_TOKEN is just a way to authorize automation.

Comment (mostaphaRoudsari) [reply .travis.yml]: @AntoineDao, Thank you for the comments. See the updated version!

Comment (AntoineDao) [reply .travis.yml]: Nice one!

**Comment by AntoineDao** [comment]
I'll let you do the honours of merging it 👍

**Comment by mostaphaRoudsari** [comment]
I'm going to merge this PR now! Fingers crossed it will work.

**Comment by chriswmackey** [comment]
🎉 !!!!!!

**Comment by AntoineDao** [comment]
Totally worked the first time and from this PR... not the 5 other PRs and test configs on Travis we ran through subsequently 🙈

**Pull request merged**